using-timescaledb/compression.md

```diff
@@ -211,18 +264,19 @@ SELECT show_chunks('conditions', older_than => interval '3 days');
 |1|_timescaledb_internal_hyper_1_2_chunk|
 |2|_timescaledb_internal_hyper_1_3_chunk|
 
-
 From here we can begin the process of compressing each of the listed chunks with the
 following command:
 
 ``` sql
 SELECT compress_chunk( '<chunk_name>');
 ```
+
 You can see the results of the compression of that given chunk by running the following:
 
 ``` sql
-SELECT * from _timescaledb_catalog.compression_chunk_size
-  ORDER BY chunk_id;
+SELECT * 
+FROM timescaledb_information.compressed_chunk_stats
+ORDER BY chunk_id;
 ```
 This will return a result set that will show you the compressed chunks and the
 stats about those chunks.
```

Comment:
Can you show what the output of this command is?

Also, include a query (and output) for: list of all uncompressed chunks?